dolphin/task_manager/tasks/task.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2020 The SODA Authors.
```

Comment:
is tasks folder required if define task.py? how about the /tasks/resources.py, so that the developer can and tasks/performances.py in the future. otherwise, IMO, tasks folder is not required

Reply:
for now, it's task. later can be added and renamed based on new functionalities